### PR TITLE
Feat/34 studio page

### PR DESCRIPTION
**Pull request by jiayuu-u**
### Descriptions

- Link to UI design (if possible):

image

### Provide screenshots or videos that the function is working.
image
image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Comment by tatoMa** [comment]
First question, how can we enter this page? By clicking where?

**Comment by jiayuu-u** [comment]
> First question, how can we enter this page? By clicking where?

I have added the link to the nav bar and we can click the icon to enter the home page

**Review by HarryuCATGREY — APPROVED** [review]
Learn a lot, LGTM

**Comment by HarryuCATGREY** [comment]
> Learn a lot, LGTM

File: components/general/Contact.tsx [hunk 1]
```diff
@@ -4,11 +4,10 @@ const Contact = () => {
...
     <InfoBlock
       sectionName="Contact"
-      title="Let’s bring your brand to the next level"
-      iconPath="/images/arrows.svg"
+      title="Let’s bring your tech to the next level"
+      iconPath="/images/design_icon.svg"
       blockType="normal"
-      content="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
-          ut labore et dolore magna aliqua."
+      content="Contact us to explore partnership opportunities and discover how we can work together to drive innovation and create positive change."
```
Comment (tatoMa) [inline components/general/Contact.tsx:10]: Nice work!!!!!!

**Review by tatoMa — COMMENTED** [review]
2 different versions of OurValues pages?

**Comment by jiayuu-u** [comment]
> 2 different versions of OurValues pages?

There is two different component of OurValues section on Figma.

**Comment by Cameron933** [comment]
image
Perhaps it would be better to give this subsection another name?

**Comment by jiayuu-u** [comment]
> image Perhaps it would be better to give this subsection another name?

Do you mean the ItemList component?

**Review by Cameron933 — APPROVED** [review]
LGTM so far

**Pull request merged**